swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category=None, condition=0):
+        if not category:
+            category = ""
+        self.category = category
```

Comment:
For `Item`'s default parameters, strings and numbers are immutable, so we don't need to handle setting the values the same way we need to handle a mutable parameter, like `inventory` on the `Vendor` class. We could directly use `""` as  the default value for `category` here.
```suggestion
    def __init__(self, category="", condition=0):
        self.category = category
```